examples/asr/conf/config.yaml

```diff
@@ -166,6 +166,8 @@ pl:
     num_nodes: 1
     distributed_backend: ddp
     accumulate_grad_batches: 1
+    amp_level: O0
+    precision: 32
```

Comment:
Why did you make this change?

Reply:
Isnt this required for AMP? I will revert it then.

Reply:
Why is the default 32-bit and not 16 bit? I think `amp_level` is deprecated so let's drop that. Let's also default gpus to 1, so our default of 16 and gpus=0 doesn't crash

Reply:
Reverted it entirely for now as  these arguments are not the core focus. Will send another PR to correct this later.